.github/actions/setup-c8run/action.yml

```diff
@@ -136,7 +136,7 @@ runs:
       if: ${{ inputs.source-build == 'true' }}
       working-directory: ./operate/client
       shell: bash
-      run: yarn
+      run: npm ci
```

Comment:
Please also remove the unnecessary `setup-yarn-cache` step for operate in lines 125-128

Reply:
@cmur2 Done